app.yaml

```diff
@@ -1,8 +1,12 @@
 runtime: python27
-threadsafe: true
+threadsafe: false
```

Comment:
https://cloud.google.com/appengine/docs/standard/python/migrate-to-python3/config-files#updating_appyaml. This field is depreciated in python 3 and `All applications are presumed to be threadsafe`.

Reply:
I think of this as an experiment to resolve a current operational problem even if it cannot be the long-term solution.
Hopefully, ndb works faster in py3.  If not, we can look into a new way to access datastore.

Reply:
Ack